micro-rdk-server/Cargo.toml

```diff
@@ -50,8 +50,8 @@ path = "esp32/main.rs"
 
 
 [package.metadata.esp-idf-sys]
+esp_idf_version = "v4.4.4"
```

Comment:
By setting the ESP-IDF version this way, we get the default of v4.4.4 (matches canon image) but it can be overridden via an environment variable.